jobs/build_ova/build_ova.sh

```diff
@@ -1,24 +1,34 @@
-#!/bin/bash
+#!/bin/bash -x
+
 set +e
-#sudo cp ${HOME}/bin/packer   /usr/bin
-#sudo apt-get install -y  jq
+mv $WORKSPACE/cache_image/packer/* $WORKSPACE/build/packer/
```

Comment:
I noticed there are *.md5 file under packer/ directory, Is it feasible to check the md5sum before using existing images. just a thought and confirm with you